src/utilities/getBmi/getBmi.js

```diff
@@ -0,0 +1,21 @@
+export const getBmi = (weight, height) => {
```

Comment:
Please take a close look at this function and improve it.

- it has an unnecessary empty line
- the if statements can be improved
- the expected return values are incorrect